.circleci/config.yml

```diff
@@ -769,8 +769,25 @@ jobs:
             aws s3 cp ${WHEELFILE} s3://core-previews/${PREVIEW_BRANCH}/ --acl public-read
             cd ../..
 
+            echo -e "Here is the link to download the wheel file: https://core-previews.s3-us-west-2.amazonaws.com/${PREVIEW_BRANCH}/${WHEELFILE}"
             echo -e "https://core-previews.s3-us-west-2.amazonaws.com/${PREVIEW_BRANCH}/${WHEELFILE}" >> S3_URL
 
+      - run:
+          name: Upload wheel for Metrics App!
+          command: |
+            ${SUDO} apt-get install -y awscli
+            aws configure set aws_access_key_id ${CORE_PREVIEWS_S3_KEY_ID}
+            aws configure set aws_secret_access_key ${CORE_PREVIEWS_S3_SECRET_KEY}
+            cd lib/dist
+            for a in *.whl; do mv -i "$a" streamlit-11.11.11-py2.py3-none-any.whl; done 
+            export WHEELFILE="$(ls -t *.whl | head -n 1)"
+            aws s3 cp ${WHEELFILE} s3://core-previews/${PREVIEW_BRANCH}/ --acl public-read
+            echo -e "Here is the link to download the wheel file: https://core-previews.s3-us-west-2.amazonaws.com/${PREVIEW_BRANCH}/${WHEELFILE}"
```

Comment:
Using a for loop to move what should always be a single file seems pretty janky to me (note that this will also essentially delete all but the last file moved if there does happen to be >1 wheel somehow).

I'd imagine that the `aws s3 cp` should work more or less the same way as the normal unix `cp` command, which would let you write this as

```bash
export WHEELFILE="$(ls -t *.whl | head -n 1)"
aws s3 cp ${WHEELFILE} s3://core-previews/${PREVIEW_BRANCH}/streamlit-11.11.11-py2.py3-none-any.whl --acl public-read
```

so that the for loop isn't necessary.